csi/deployment.go

```diff
@@ -418,6 +418,14 @@ func NewPluginDeployment(namespace, serviceAccount, nodeDriverRegistrarImage, li
 									Name:  "CSI_ENDPOINT",
 									Value: GetCSIEndpoint(),
 								},
+								{
+									Name: "POD_NAMESPACE",
+									ValueFrom: &corev1.EnvVarSource{
+										FieldRef: &corev1.ObjectFieldSelector{
+											FieldPath: "metadata.namespace",
+										},
+									},
+								},
```

Comment:
Maybe we don't need to add namespace if we don't need to look up the share manager pod as mentioned below